cmake/Modules/OpmSatellites.cmake

```diff
@@ -1,4 +1,8 @@
 # - Build satellites that are dependent of main library
+
+option(ADD_DISABLED_CTESTS "Add the tests which are disabled due to failed preconditions to the ctest output (this makes ctest return an error if such a test is present)" ON)
+mark_as_advanced(ADD_DISABLED_CTESTS)
```

Comment:
nothing i care too much about, but i thought it was a left-over and not a conscious choice. we'll leave it as-is.